frontend/apps/vara-man/package.json

```diff
@@ -5,7 +5,7 @@
 	"version": "1.0.2",
 	"type": "module",
 	"scripts": {
-		"start": "yarn build:packages && yarn build:ez-transactions && vite --open",
+		"start": "yarn build:packages && yarn build:ez-transactions && tsc && vite --open --host",
```

Comment:
What's the point of this parameters?

Reply:
@nikitayutanov  tsx is obviously superfluous here, --host is needed to check the workability on mobile devices